### PR TITLE
5024 doc/update kubeapps on tce tutorial

### PR DESCRIPTION
**Pull request by ppbaena**
### Description of the change

Update Kubeapps on TCE tutorial to comply with style guides.

### Benefits

Documentation updated and standardize according to Kubeapps style guides based on Kubernetes style guides

### Possible drawbacks

N/A

### Applicable issues

- related to #5024

### Additional information

An index at the end of each section have been added to ease the navigation across sections.

### CODE REVIEW

File: site/content/docs/latest/tutorials/kubeapps-on-tce/02-TCE-managed-cluster.md [hunk 1]
```diff
@@ -97,8 +97,16 @@ This information will be used to both set up the Kubernetes cluster, and in a fu
    tanzu package repository add tce-repo --url projects.registry.vmware.com/tce/main:0.12.0 --namespace tanzu-package-repo-global
    ```
 
-The outcome of the actions above will be a management, managed TCE cluster running on your preferred infrastructure provider. Starting with that, _workload_ clusters could be created and be controlled from this _management_ cluster. For simplicity reasons, we will continue the tutorial using only the management cluster, but it is not a recommended strategy for production setups.
+The outcome of the actions above is a management, managed TCE cluster running on your preferred infrastructure provider. Starting with that, _workload_ clusters could be created and be controlled from this _management_ cluster. For simplicity reasons, the tutorial continues by using only the management cluster, but it is not a recommended strategy for production setups.
```
Comment (antgamdia) [inline site/content/docs/latest/tutorials/kubeapps-on-tce/02-TCE-managed-cluster.md:100]:
```suggestion
The outcome of the actions above is a _managed_ TCE cluster running on your preferred infrastructure provider. From this _management_ cluster, new _workload_ clusters can be spun up and attached to it.

For the sake of simplicity, this tutorial only uses a single _management_ cluster with no attached _workload: clusters, but note that it is not a recommended strategy for production setups.
```

File: site/content/docs/latest/tutorials/kubeapps-on-tce/02-TCE-unmanaged-cluster.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Step 2A: Deploy a VMware Tanzu™ Community Edition unmanaged cluster
 
-In this step of the tutorial, we will install an [unmanaged TCE cluster](https://tanzucommunityedition.io/docs/v0.12/planning/#unmanaged-cluster).
+In this step of the tutorial, the outcome is to install an [unmanaged TCE cluster](https://tanzucommunityedition.io/docs/v0.12/planning/#unmanaged-cluster).
```
Comment (antgamdia) [inline site/content/docs/latest/tutorials/kubeapps-on-tce/02-TCE-unmanaged-cluster.md:3]:
```suggestion
In this step, the goal is to install an [unmanaged TCE cluster](https://tanzucommunityedition.io/docs/v0.12/planning/#unmanaged-cluster).
```

File: site/content/docs/latest/tutorials/kubeapps-on-tce/03-preparing-kubeapps-deployment.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+Some relevant topics like routing traffic into Kubeapps, TLS, or which plugins need to be enabled, are set up in a _configuration values file_.
+A configuration values file is a Yaml file that allows you to customize the deployment of Kubeapps. TCE makes use of [Carvel](https://carvel.dev/) for installing applications, and in the case of the Kubeapps package, the configuration file uses exactly the same parameters specified in the [Bitnami Kubeapps Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps#parameters). It is highly recommended that you take a look at the possible parameters and get familiar with them.
 
-The outcome of this tutorial step will be:
+The outcome of this tutorial step is:
```
Comment (antgamdia) [inline site/content/docs/latest/tutorials/kubeapps-on-tce/03-preparing-kubeapps-deployment.md:8]:
```suggestion
The outcome of this step is:
```

File: site/content/docs/latest/tutorials/kubeapps-on-tce/README.md [hunk 4]
```diff
@@ -2,6 +2,6 @@
 
 [VMware Tanzu™ Community Edition (TCE)](https://tanzucommunityedition.io) is a full-featured, easy-to-manage Kubernetes platform for learners and users, especially those working in small-scale or preproduction environments.
 
-[Kubeapps](https://kubeapps.com/) provides a web-based dashboard to deploy, manage, and upgrade applications on a Kubernetes cluster. It is a one-time install that gives you a number of important benefits, including the ability to browse and deploy Helm charts from public or private chart repositories and upgrade, manage and delete deployments from your Web browser.
+[Kubeapps](https://kubeapps.com/) provides a web-based dashboard to deploy, manage, and upgrade applications on a Kubernetes cluster. It is a one-time install that gives you a number of important benefits, including the ability to browse and deploy Helm charts and Carvel packages from public or private repositories and upgrade, manage and delete deployments from your Web browser.
```
Comment (antgamdia) [inline site/content/docs/latest/tutorials/kubeapps-on-tce/README.md:5]: If we want to add Carvel here (or sth cooler like "any package"), we might want to replicate this change to: 
- Repo description
- Chart metadata
- Chart readme + bitnami
- Website

**Review by antgamdia — APPROVED** [review]
Thanks!

File: site/content/docs/latest/tutorials/kubeapps-on-tce/README.md (hunk 4 above)
Comment (ppbaena) [reply site/content/docs/latest/tutorials/kubeapps-on-tce/README.md:5]: Agree.

**Pull request merged**